### ISSUE TITLE
Table Row background color doesn't fill the whole row

### ISSUE THREAD

**Issue by gadzhov** [daohoangson/flutter_widget_from_html#1028]
## Steps to Reproduce

<details>
  <summary>HTML</summary>

```html
<table style="border-collapse: collapse;">
  <tr>
    <th>First Name</th>
    <th>Last Name</th>
    <th>Points</th>
  </tr>
  <tr style="background-color: #f2f2f2;">
    <td>Jill</td>
    <td>Smith</td>
    <td>50</td>
  </tr>
  <tr>
    <td>Eve</td>
    <td>Jackson</td>
    <td>94</td>
  </tr>
  <tr style="background-color: #f2f2f2;">
    <td>Adam</td>
    <td>Johnson</td>
    <td>67</td>
  </tr>
</table>
```
</details>

<details>
  <summary>`HtmlWidget` configuration</summary>

```dart
 HtmlWidget(html);
```
</details>

<details>
  <summary>Tesing environment</summary>

```

[√] Flutter (Channel stable, 3.7.6, on Microsoft Windows [Version 10.0.22621.2134], locale bg-BG)
    • Flutter version 3.7.6 on channel stable at D:\tools\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 12cb4eb7a0 (7 months ago), 2023-03-01 10:29:26 -0800
    • Engine revision ada363ee93
    • Dart version 2.19.3
    • DevTools version 2.20.1

[√] Windows Version (Installed version of Windows is version 10 or higher)

[√] Android toolchain - develop for Android devices (Android SDK version 33.0.2)
    • Android SDK at C:\Users\user\AppData\Local\Android\sdk
    • Platform android-33-ext4, build-tools 33.0.2
    • Java binary at: C:\Program Files\Android\Android Studio\jbr\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-9505619)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[X] Visual Studio - develop for Windows
    X Visual Studio not installed; this is necessary for Windows development.
      Download at https://visualstudio.microsoft.com/downloads/.
      Please install the "Desktop development with C++" workload, including all of its default components

[√] Android Studio (version 2022.1)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.15+0-b2043.56-9505619)

[√] VS Code (version 1.82.0)
    • VS Code at C:\Users\user\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.72.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22621.2134]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 114.0.5735.134
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 116.0.1938.76

[√] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```
</details>

## Expected results

![image](https://github.com/daohoangson/flutter_widget_from_html/assets/18680832/9ce25dcf-55c0-4a78-9c69-1ac27a4da80e)

## Actual results

![image](https://github.com/daohoangson/flutter_widget_from_html/assets/18680832/914918a2-811e-464f-8386-92f90ade6ede)

**Comment by daohoangson** [comment]
This has been fixed in the pre-release version `0.13.1-alpha.1`. The fix cannot be back ported to the stable version (v0.10 series). Can you try running your app with the alpha version to verify the fix?

**Comment by daohoangson** [comment]
Good news! The latest stable (`v0.14.2`) has been released with the fix for table row background.
